### ISSUE TITLE
Alpha channel for decoration plugin not consistent

### ISSUE THREAD

**Issue by xbjfk** [WayfireWM/wayfire#1258]
**Describe the bug**
It seems that the alpha channel is for the decoration plugin does not work as it should

**To Reproduce**
1. For example, set the color to #FFFFFF00
2. And observe how the decoration color is not transparent.

**Expected behavior**
The behavior should be consistent with other apps - an alpha value of 00 should always be transparent, 80 half transparent etc.

**Screenshots or stacktrace**
For example - take the color value #FF000000 - it is expected to be fully transparent, however, it looks like this:
![image](https://user-images.githubusercontent.com/24711088/128837703-4421986d-7a38-4bc1-96aa-0f0175ea089c.png)

In a more practical example, this makes it completely impossible to use a light mode decoration. The following screenshot uses the value #FFFFFF80 
![image](https://user-images.githubusercontent.com/24711088/128838079-b3522af9-8983-4a75-b48d-698759475f5a.png)

**Wayfire version**
0.8.0-35ff4fda (Aug 10 2021, branch 'HEAD')

**Comment by ammen99** [comment]
Yeah, I see what you mean. In general, buffers and colors in Wayfire have premultiplied alpha, but I guess users expect not premultiplied alpha.

For a quick workaround, try multiplying the RGB values by the alpha value, it should give you the effect you want.

To fix this permanently, we can do this multiplication in the rendering code.

**Comment by ammen99** [comment]
I feel like changing the behavior now will only break config files. Wayfire simply expects premultiplied alpha and users can do that in their config files.
